src/content/en/_shared/udacity/ud811.html

```diff
@@ -1,4 +1,4 @@
-<div itemscope itemtype="http://schema.org/Course">
+<div itemscope itemtype="http://schema.org/Course" dir="ltr">
```

Comment:
Some RTL translated docs doesn't have translated _shared/udacity files, so I added dir here to avoid floating from RTL in these langs!